theano/gpuarray/opt.py

```diff
@@ -1217,8 +1175,8 @@ def local_gpua_softmaxwithbias(node, context_name):
 @register_opt2([theano.tensor.opt.Assert], 'fast_compile')
 def local_assert(op, context_name, inputs):
     return [host_from_gpu(op(as_gpuarray_variable(inputs[0],
```

Comment:
I've done this in entirety here, https://github.com/Theano/Theano/pull/4672/commits/5a9c296d5fd524fe29e149365a79fe8e8d924cc4